api.yaml

```diff
@@ -0,0 +1,135 @@
+openapi: 3.0.0
+x-runtime:
+  httpserver:
+    address: ":8080"
+  logger:
+    level: "INFO"
+    output: "STDOUT"
+  stats:
+    output: "DATADOG"
+    datadog:
+      address: "statsd:8125"
+      flushinterval: "10s"
+  signals:
+    installed:
+      - "OS"
+    os:
+      signals:
+        - 2 # SIGINT
+        - 15 # SIGTERM
+  connstate:
+    reportinterval: "5s"
+    hijackedcounter: "http.server.connstate.hijacked"
+    closedcounter: "http.server.connstate.closed"
+    idlegauge: "http.server.connstate.idle.gauge"
+    idlecounter: "http.server.connstate.idle"
+    activegauge: "http.server.connstate.active.gauge"
+    activecounter: "http.server.connstate.active"
+    newgauge: "http.server.connstate.new.gauge"
+    newcounter: "http.server.connstate.new"
+x-transportd:
+  backends:
+    - app
+  app:
+    host: "http://app:8081"
+    pool:
+      ttl: "24h"
+      count: 1
+info:
+  version: 1.0.0
+  title: "Nexpose Scan Notifier"
+  description: "API service which queries Nexpose and generates events for completed scans"
+  contact:
+    name: Security Development
+    email: secdev-external@atlassian.com
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+paths:
+  /healthcheck:
+    get:
+      description: "Liveness check."
+      responses:
+        "200":
+          description: "Success."
+      x-transportd:
+        backend: app
+  /notification:
+    post:
+      description: Poll Nexpose for scans completed within the provided duration
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NotificationRequest'
```

Comment:
@faldridge I took a stab, but please let me know if this is what you were thinking for the `/notification` API.

Reply:
Yep, that is exactly what I was thinking. Thanks!! 👍